.github/workflows/publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish
+on:
+  release:
```

Comment:
As written, this triggers when someone goes and makes a "release" on github.

This doesn't address the flow up to the point where we make a release. More notes on that shortly.